scene/register_scene_types.cpp

```diff
@@ -712,6 +712,11 @@ void register_scene_types() {
 	String font_path = GLOBAL_DEF("gui/theme/custom_font", "");
 	ProjectSettings::get_singleton()->set_custom_property_info("gui/theme/custom_font", PropertyInfo(Variant::STRING, "gui/theme/custom_font", PROPERTY_HINT_FILE, "*.tres,*.res,*.font", PROPERTY_USAGE_DEFAULT | PROPERTY_USAGE_RESTART_IF_CHANGED));
 
+	if (Engine::get_singleton()->is_editor_hint()) {
```

Comment:
I would probably put this in EditorNode::register_editor_types

Reply:
I amended the commit accordingly.